modular_citadel/code/_DEFINES/machines.dm

```diff
@@ -0,0 +1,2 @@
+#define	AUTOYLATHE	256	//Uses glass/metal/Plastic
```

Comment:
you defined it earlier and here, why

Reply:
Same as the other one. Did leave the note though